index.html

```diff
@@ -6,15 +6,173 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital@0;1&display=swap" rel="stylesheet">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <!-- Header -->
+    <header class="container">
+        <div class="row">
+            <nav class="navbar navbar-expand-sm bg-white navbar-light fixed-top">
+               
+                <div class="col-12 col-sm-8 order-sm-2 justify-content-around d-flex flex-sm-column align-items-sm-end"> 
+                    <a class="col-8 col-sm-5 col-md-4 col-lg-2 appointment-button bg-lightmint text-decoration-none text-center" href="#">Book an Appointment</a>
```

Comment:
Dont' use columns inside of columns without using a new `row` in between them as it will mess with the alignment of elements.